BrowserStack.java

```diff
@@ -34,36 +34,83 @@ public static void myassert(boolean  x) {
             throw new IllegalArgumentException("Assert fail") ;
         }
     }
+    
     //push() - pushes the element in this case its a url (String)
     private  void push(String url){
-        //WRITE YOUR CODE HERE
+        top++;
```

Comment:
Stack was initialized to size of 16. So, we should check for Stack full condition and return appropriate error.